assets/js/recruiter-box.js

```diff
@@ -109,9 +109,19 @@ class RecruiterBox {
         // TODO handle select
       }
 
+      const fieldnames = ['firstName', 'lastName', 'email', 'phone', '_gotcha'].concat(customFields);
```

Comment:
![Screenshot](https://github.com/c4a8-web/shared-components/assets/4637866/90015748-f4af-448f-88df-d603422631d2)

Sehen die Formfelder nach wie vor so aus? also die CustomFields haben ja vorher keinen Prefix "candidate" bekommen. Aufgrund der Tatsache, dass wir heute auch den Use-Case neuer Felder haben, sollten wir vielleicht nur die elemente, die wir kennen prefixen, also alles andere ist ein customField ohne prefix. Dann würden auch so Felder die neu dazu kommen, einfach behandelt und durchgereicht werden. Wie "portfolio" in meinem Screenshot.